src/Modal.js

```diff
@@ -2,8 +2,6 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import {
   Animated,
-  DeviceEventEmitter,
-  Dimensions,
   Easing,
   Modal as ReactNativeModal,
```

Comment:
Please notice that this is not https://github.com/react-native-modal/react-native-modal

Reply:
Sorry, I don't notice that 😅 The misunderstanding happens due to the definition of library [types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native-dialog/index.d.ts).